src/algorithms/community/modularity.jl

```diff
@@ -203,63 +202,3 @@ function findcommunities(h::Hypergraph, method::CFModularityCNMLike)
     end
     return (bm=best_modularity, bp=comms, mod_history=mod_history)
 end
-
```

Comment:
@aleant93 why do we want to remove this function?
Looks like some large work being done.

Reply:
We removed this functionality as we found that it was implemented in the `Clustering.jl` Julia package, defined exactly as we used it. Here's the link to the [function](https://juliastats.org/Clustering.jl/stable/validate.html#Mutual-information-1).